.changeset/dry-wolves-join.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-search': patch
+'@backstage/plugin-search-react': patch
+---
+
+Fix issue with `HomePageSearchBar` requiring `SearchContext`
```

Comment:
Maybe worth splitting the two changesets up so that you can write for `plugin-search-react` that it now exports a hook to check if the search context is available. And then keep this line just for `plugin-search`. What do you think?

Reply:
@kuangp If you fix the changeset you can also prefix with `search-*`

Reply:
done!